wcomponents-core/src/main/java/com/github/bordertech/wcomponents/WMultiFileWidget.java

```diff
@@ -208,16 +229,27 @@ public void setFileTypes(final String[] types) {
 
 	/**
 	 * Determines the file types accepted by this widget. Note that duplicates are not allowed and these are not case
-	 * sensitive. A file type may be one of the following:
+	 * sensitive.<br>
+	 * The file type(s) can be either:
 	 * <ul>
-	 * <li>The string audio/* (Indicates that sound files are accepted.)</li>
```

Comment:
Restore the original text of this list as it is **more correct** than that which replaced it.